examples/yaml-filebeat/filebeat.yml

```
@@ -0,0 +1,28 @@
+filebeat.inputs:
+- type: unix
+  id: "NDPId-logs" # replace this index to your preference
+  max_message_size: 100MiB
+  index: "index-name" # Replace this with your desired index name in Elasticsearch
+  enabled: true
+  path: "/var/run/nDPId.sock" # point nDPId to this Unix Socket
```

I guess it's the collector socket you're referring to, right?

```suggestion
  path: "/var/run/nDPId.sock" # point nDPId to this Unix Socket (Collector)
```

---

Yes this is the collector socket